docs/docsite/rst/dev_guide/developing_plugins.rst

```diff
@@ -246,7 +246,7 @@ Connection plugins allow Ansible to connect to the target hosts so it can execut
 
 Ansible version 2.1 introduced the ``smart`` connection plugin. The ``smart`` connection type allows Ansible to automatically select either the ``paramiko`` or ``openssh`` connection plugin based on system capabilities, or the ``ssh`` connection plugin if OpenSSH supports ControlPersist.
 
-To create a new connection plugin (for example, to support SNMP, Message bus, or other transports), copy the format of one of the existing connection plugins and drop it into the ``connection_plugins`` directory.
+To create a new connection plugin (for example, to support SNMP, Message bus, or other transports), copy the format of one of the existing connection plugins and drop it into the ``plugins/connection`` directory.
```

Comment:
```suggestion
To create a new connection plugin (for example, to support SNMP, Message bus, or other transports), copy the format of one of the existing connection plugins and drop it into ``connection`` directory on your :ref:`local plugin path <local_plugins>`.
```

Reply:
That works for me. It seems to clarify while staying accurate.

Reply:
@kbreit you can incorporate this change into your PR by accepting the suggestion.